src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/message/processors/prison/BasePrisonEventProcessor.kt

```diff
@@ -0,0 +1,30 @@
+package uk.gov.justice.digital.hmpps.personrecord.message.processors.prison
+
+import kotlinx.coroutines.runBlocking
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Component
+import uk.gov.justice.digital.hmpps.personrecord.client.PrisonerSearchClient
+import uk.gov.justice.digital.hmpps.personrecord.client.model.prisoner.Prisoner
+import uk.gov.justice.digital.hmpps.personrecord.service.RetryExecutor
+
+@Component
+class BasePrisonEventProcessor(
+  private val prisonerSearchClient: PrisonerSearchClient,
+
+  private val retryDelay: Long = 0,
```

Comment:
Needs the @Value annotation as its not getting populated from env vars currently